package.json

```diff
@@ -517,243 +517,239 @@
+          "command": "zowe.editMember",
+          "group": "6_modification@2"
         },
         {
-          "when": "view == zowe.uss.explorer && viewItem =~ /^(?!.*_fav.*)ussSession.*/",
-          "command": "zowe.all.profilelink",
-          "group": "inline"
+          "when": "view == zowe.explorer && viewItem =~ /^member.*/",
+          "command": "zowe.renameDataSetMember",
+          "group": "6_modification@3"
         },
         {
-          "when": "viewItem =~ /^(?!.*_fav.*)ussSession.*/",
-          "command": "zowe.uss.fullPath",
-          "group": "inline"
+          "when": "view == zowe.explorer && viewItem =~ /^ds.*/",
+          "command": "zowe.renameDataSet",
+          "group": "6_modification@3"
         },
         {
-          "when": "viewItem =~ /^(?!.*_fav.*)ussSession.*/",
-          "command": "zowe.uss.createNode",
-          "group": "inline"
+          "when": "view == zowe.explorer && viewItem =~ /^pds.*/",
+          "command": "zowe.renameDataSet",
+          "group": "6_modification@3"
         },
         {
-          "when": "viewItem =~ /^(?!.*_fav.*)ussSession.*/",
-          "command": "zowe.uss.editSession",
-          "group": "inline"
+          "when": "view == zowe.explorer && viewItem =~ /^ds.*/",
+          "command": "zowe.deleteDataset",
+          "group": "6_modification@4"
         },
         {
-          "when": "viewItem =~ /^(?!.*_fav.*)ussSession.*/",
-          "command": "zowe.uss.deleteProfile",
+          "when": "view == zowe.explorer && viewItem =~ /^member.*/",
+          "command": "zowe.deleteMember",
           "group": "6_modification@4"
         },
         {
-          "when": "viewItem =~ /^(?!.*_fav.*)ussSession.*/",
-          "command": "zowe.uss.removeSession",
-          "group": "6_modification@3"
+          "when": "view == zowe.explorer && viewItem =~ /^pds.*/",
+          "command": "zowe.deletePDS",
+          "group": "6_modification@4"
         },
         {
-          "when": "view == zowe.uss.explorer && viewItem =~ /^(?!.*_fav.*)ussSession.*/",
-          "command": "zowe.uss.saveSearch",
-          "group": "4_workspace"
+          "when": "view == zowe.explorer && viewItem =~ /^(?!.*_fav.*)session.*/",
+          "command": "zowe.deleteProfile",
+          "group": "6_modification@5"
         },
         {
-          "when": "view == zowe.uss.explorer && viewItem =~ /^(?!.*_fav.*)ussSession.*/",
-          "command": "zowe.issueMvsCmd",
-          "group": "3_systemSpecific"
+          "when": "view == zowe.explorer && viewItem =~ /^(ds.*|^pds.*)/",
+          "command": "zowe.hMigrateDataSet",
+          "group": "7_hsmCommands"
         },
         {
-          "when": "view == zowe.jobs && viewItem =~ /^(?!.*_fav.*)server.*/",
-          "command": "zowe.all.profilelink",
-          "group": "inline"
+          "when": "view == zowe.explorer && viewItem =~ /^migr.*/",
+          "command": "zowe.hRecallDataSet",
+          "group": "8_hsmCommands"
```

Comment:
Hey @katelynienaber could you change the `8_hsmCommands` to `7_hsmCommands@0`? 2 reasons for this, first the `Recall data set` is only shown when the data set is migrated, so probably better to have in the same group, also the `@0` is because ill be adding a `hDelete` for deleting migrated data sets soon, in which I will put it in the group `7_hsmCommands@1`.